solutions/product_solutions/Techex_TX_Core/01_techex-tx-core-install.md

```diff
@@ -0,0 +1,47 @@
+---
+uid: techex-tx-core-install
+---
+
+# Installing the Techex TX Core app
+
+## Prerequisites
+
+- DataMiner license [Tier 1](xref:Pricing_Perpetual_Use_Licensing) or [subscription](xref:Pricing_Usage_based_service)
+- DataMiner 10.4.8/10.5.0 or higher
+- DataMiner System using [STaaS](xref:STaaS) or [dedicated clustered storage](xref:Dedicated_clustered_storage)
+- The [GenericInterface](xref:Overview_of_Soft_Launch_Options#genericinterface) soft-launch option must be enabled.
+- [Techex MWCore connector](https://catalog.dataminer.services/details/838c9515-69fd-4405-9284-822cb8bd5686)
+
+## Deploying the DataMiner Techex TX Core app
+
+1. Look up the [*Techex TX Core* package](https://catalog.dataminer.services/details/d9ec570f-a625-40c1-a6fa-c9b4f15416cd) in the Catalog.
```

Comment:
This links to the NetInsight Nimbra Vision package. Could you correct this link so it leads to the right package?